src/main/kotlin/org/malachite/estella/interview/api/InterviewController.kt

```diff
@@ -0,0 +1,64 @@
+package org.malachite.estella.interview.api
+
+import org.malachite.estella.commons.OwnResponses
+import org.malachite.estella.commons.models.interviews.InterviewNote
+import org.malachite.estella.interview.domain.InterviewId
+import org.malachite.estella.services.InterviewService
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.http.ResponseEntity
+import org.springframework.transaction.annotation.Transactional
+import org.springframework.web.bind.annotation.*
+import java.sql.Timestamp
+import java.util.*
+import javax.sql.rowset.serial.SerialClob
+
+@RestController
+@RequestMapping("/api/interview")
+class InterviewController(
+        @Autowired private val interviewService: InterviewService
+) {
+
+    @CrossOrigin
+    @GetMapping("/jobseeker/{interviewId}")
+    fun getJobseekerName(@PathVariable interviewId: InterviewId): ResponseEntity<JobseekerName> =
+        interviewService.getUserFromInterviewUuid(interviewId.toUUID())
+                .let {
+                    ResponseEntity.ok(JobseekerName(it!!.firstName, it.lastName))
```

Comment:
Nie lepiej dać nullowalne firstName? Jeśli ktoś nie poda (jest w ogóle taka możliwość?), to wtedy to rzuci wyjątkiem, a lepiej mieć niepełne, ale coś, aniżeli nie mieć nic.